staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator.go

```diff
@@ -278,6 +293,30 @@ func (s *specAggregator) UpdateAPIServiceSpec(apiServiceName string, spec *spec.
 	})
 }
 
+// AddUpdateLocalAPIService allows adding/updating local API service with nil handler and
+// nil Spec.Service. This function can be used for local dynamic OpenAPI spec aggregation
+// management (e.g. CRD)
+func (s *specAggregator) AddUpdateLocalAPIServiceSpec(name string, spec *spec.Swagger, etag string) error {
+	s.rwMutex.Lock()
+	defer s.rwMutex.Unlock()
+
+	localAPIService := apiregistration.APIService{
+		Spec: apiregistration.APIServiceSpec{
+			Service:              &apiregistration.ServiceReference{},
+			Group:                "dynamiclocalgroup",
+			Version:              "v1",
+			GroupPriorityMinimum: 1000, // CRDs should have relatively low priority
```

Comment:
the kube-aggregator should not know about CRD priorities. Turn that into argument of this func.

Actually I don't like the whole artificial APIService object here.

Reply:
this function was not used. Had it here because two aggregators (one in kube-aggregator and one in apiextensions-apiserver) shared the same interface somehow. Sorry for the misleading. I will clean it up